### ISSUE TITLE
hey @ndev1991

### ISSUE THREAD

**Issue by DevTalent1121** [ndev1991/empty#1]

**Comment by ndev1991** [comment]
hey!

**Comment by DevTalent1121** [comment]
any update?

**Comment by ndev1991** [comment]
hey! what update do you mean?

**Comment by DevTalent1121** [comment]
i mean new job.

**Comment by ndev1991** [comment]
currently we are discussing the future of the projects and nothing to do for now. but will be have jobs soon. anyway please send me the resume when ready.

**Comment by ndev1991** [comment]
i guess there is no update there as well? 😄  right?

**Comment by ndev1991** [comment]
i mean no movement or rumors. haha

**Comment by ndev1991** [comment]
hi?

**Comment by DevTalent1121** [comment]
Hiiiii, ndev1991

**Comment by DevTalent1121** [comment]
Long time no see!!!
How are you?

**Comment by ndev1991** [comment]
howdy, howdy! it's very nice to meet you! how are you doing?

**Comment by DevTalent1121** [comment]
Hi, how are you? Long time no see. When are you available?

**Comment by ndev1991** [comment]
hey! it's very nice to see you too! this morning i'm little too busy. how about 2am your time? that's 1am here

**Comment by DevTalent1121** [comment]
okay, I will wait for your message.

**Comment by ndev1991** [comment]
great. thanks

**Comment by ndev1991** [comment]
hey! so how's it going? it's like what happened? everything good now?

**Comment by DevTalent1121** [comment]
good. thx. how about you?

**Comment by ndev1991** [comment]
yeah, i'm good. still alive. kk

**Comment by DevTalent1121** [comment]
hh~ Im glad to hear that.

**Comment by ndev1991** [comment]
yup. so what happened? investigation finished?

**Comment by DevTalent1121** [comment]
yeah, but nothing remarkable. I need new job. Do you have any one for me?

**Comment by ndev1991** [comment]
hehe. i'm sorry, but I don't have one now

**Comment by DevTalent1121** [comment]
is that so? then what are you working on?

**Comment by ndev1991** [comment]
just waiting. static robot

**Comment by ndev1991** [comment]
so how's your family? all good?

**Comment by DevTalent1121** [comment]
Yeah, all fine, thanks. how about you? and how is your health?

**Comment by ndev1991** [comment]
yeah. i'm fine. thanks

**Comment by DevTalent1121** [comment]
there are you guys doing well?

**Comment by ndev1991** [comment]
yup, we're just fine.

**Comment by DevTalent1121** [comment]
okay, good.

**Comment by ndev1991** [comment]
just counting down. not sure what counts, but just wait for that day

**Comment by ndev1991** [comment]
one day. kk

**Comment by ndev1991** [comment]
like fairytale

**Comment by DevTalent1121** [comment]
hh~😂

**Comment by ndev1991** [comment]
i guess, no updates there until now, right?

**Comment by DevTalent1121** [comment]
yeah, right

**Comment by ndev1991** [comment]
okay, that's great

**Comment by DevTalent1121** [comment]
what are you gonna do?

**Comment by ndev1991** [comment]
hmm... what do you mean?

**Comment by DevTalent1121** [comment]
are you gonna find new job continuously?

**Comment by ndev1991** [comment]
hehe . nothing on my plate. to-do list. nothing.

**Comment by DevTalent1121** [comment]
what do you mean?

**Comment by ndev1991** [comment]
maybe trying to start find a new job, but not sure. need to be prepared myself. mind, mental etc. kk

**Comment by ndev1991** [comment]
i'm not sure. i'm doing nothing actually. kk

**Comment by ndev1991** [comment]
sorry, will be back after 30 mins, okay?

**Comment by DevTalent1121** [comment]
right, if you find many, please let me know and share with me. okay?

**Comment by ndev1991** [comment]
hehe. if i do then surely will do. but don't expect too much. i'm totally mental and depressed. no power to work again. kk

**Comment by ndev1991** [comment]
1 hour is a day and 1 day is 1 month for me now. very very boring and tired.

**Comment by DevTalent1121** [comment]
hh~ Don't worry, keep your chin up!!

**Comment by ndev1991** [comment]
kk. thanks.
